packages/cli/src/load-nodes-and-credentials.ts

```diff
@@ -174,6 +174,29 @@ export class LoadNodesAndCredentials {
 		return isContainedWithin(loader.directory, filePath) ? filePath : undefined;
 	}
 
+	resolveSchema({
+		node,
+		version,
+		resource,
+		operation,
+	}: {
+		node: string;
+		version: string;
+		resource?: string;
+		operation?: string;
+	}): string | undefined {
+		const nodePath = this.known.nodes[node]?.sourcePath;
+		if (!nodePath) {
+			return undefined;
+		}
+
+		const nodeParentPath = path.dirname(nodePath);
+		const schemaPath = ['__schema__', `v${version}`, resource, operation].filter(Boolean).join('/');
```

Comment:
some nodes in rarer cases would have operation but not resource, should we consider it here?